test/check-application

```diff
@@ -555,6 +559,75 @@ class TestApplication(testlib.MachineCase):
         b.wait_text("#containers-containers tbody tr:contains('busybox:latest') > td:nth-child(3)", "")
         b.wait_text("#containers-containers tbody tr:contains('busybox:latest') > td:nth-child(4)", "")
 
+    def testCheckpointRestore(self):
+        b = self.browser
+        m = self.machine
+
+        # Set machine to cgroup1 and reboot
+        self.execute(True, 'grubby --update-kernel=ALL --args="systemd.unified_cgroup_hierarchy=0"; reboot')
+        m.wait_reboot()
+
+        self.login_and_go("/podman", authorized=True, superuser=True)
+        b.wait_present("#app")
+        self.filter_containers('all')
+
+        # Run a container
+        self.execute(True, "podman run -dit --name swamped-crate busybox sh; podman stop swamped-crate")
+        b.wait(lambda: self.execute(True, "podman ps --all | grep -e swamped-crate -e Exited"))
+
+        b.wait_present("#containers-containers")
```

Comment:
This line is not needed, since the line underneath it is superset of this selector.